Library/Homebrew/cmd/audit.rb

```diff
@@ -186,6 +186,7 @@ def audit_file
       [/^  keg_only/,                      "keg_only"],
       [/^  option/,                        "option"],
       [/^  depends_on/,                    "depends_on"],
+      [/^  (go_)?resource/,                "resources"],
```

Comment:
A true nitpick, but I think it's better to use `resource` as the label. `option` and `depends_on` can also have multiple instances, but they are not pluralized. (Copied from my previous comment in case you missed it. Feel free to ignore if you intentionally didn't change that.)